app/android/app_template/src/org/xwalk/app/template/AppTemplateActivity.java

```diff
@@ -15,12 +15,25 @@
 import org.xwalk.app.XWalkRuntimeActivityBase;
 
 public class AppTemplateActivity extends XWalkRuntimeActivityBase {
+    private final int SYSTEM_UI_OPTIONS = View.SYSTEM_UI_FLAG_LAYOUT_STABLE |
+            View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION |
+            View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN |
+            View.SYSTEM_UI_FLAG_HIDE_NAVIGATION |
+            View.SYSTEM_UI_FLAG_FULLSCREEN |
+            View.SYSTEM_UI_FLAG_IMMERSIVE_STICKY;
+
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
     }
 
     @Override
+    public void onResume() {
+        super.onResume();
+        enterFullscreen();
```

Comment:
In activity of your test app, add ```onResume``` and call ```enterFullscreen```? Do you thinks it's no necessary to add ```onResume``` in AppTemplateActivity.java? Thanks.